cirq/protocols/json_test_data/CliffordState.json_inward

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
why did you choose to add an inward object? For backwards compatibility purposes? 
btw the below code fails - there is something odd happening in the `__str__` of the CH form class:

<details>

```
import cirq

j = """{
  "cirq_type": "CliffordState",
  "qubit_map": [
    [
      {
        "cirq_type": "LineQubit",
        "x": 0
      },
      0
    ]
  ],
  "tableau": {
    "cirq_type": "CliffordTableau",
    "n": 1,
    "rs": [
      false,
      false,
      false
    ],
    "xs": [
      [
        true
      ],
      [
        false
      ],
      [
        false
      ]
    ],
    "zs": [
      [
        false
      ],
      [
        true
      ],
      [
        false
      ]
    ]
  },
  "ch_form": {
    "cirq_type": "StabilizerStateChForm",
    "n": 1,
    "G": [
      [
        true
      ]
    ],
    "F": [
      [
        true
      ]
    ],
    "M": [
      [
        false
      ]
    ],
    "gamma": [
      0
    ],
    "v": [
      false
    ],
    "s": [
      false
    ],
    "omega": 1
  }
}
"""

state = cirq.read_json(json_text=j)
print(state)
```

</details>

Reply:
Yeah, the inward object is for backwards compatibility since the json representation has changed by removing the tableau. I think this is the standard practice, had to do similar changes in https://github.com/quantumlib/Cirq/pull/2874/files. 

Following is the error I got before making any json changes:

```
E           AssertionError: The json produced by cirq no longer agrees with the json in the cirq/protocols/json_test_data/CliffordState.json test data file.
E             
E             You must either fix the cirq code to continue to produce the same output, or you must move the old test data to cirq/protocols/json_test_data/CliffordState.json_inward and create a fresh cirq/protocols/json_test_data/CliffordState.json file.
```

Great catch regarding the StabilizerStateChForm parsing issue! Fix in #3679

Reply:
Got it, cool, thanks for the explanation!